mongo.go

```diff
@@ -35,11 +36,23 @@ func (*Mongo) NewClient(connURI string) interface{} {
 	if err != nil {
 		return err
 	}
-	return &Client{client: client}
 
+		return  &Client{client: client}
 }
 
-func (c *Client) Insert(database string, collection string, doc interface{}) error {
+func VerifyITCollection(collection string) error {
+    log.Printf("Checking if collection name is as per requirement or not.")
+    if strings.HasPrefix(collection, "test") == true {
+      log.Printf("The collection name is as per the requirement.")
+      return nil
+   } else {
+       log.Printf("The collection name is invalid. Must begin with : test")
+       return errors.New("The collection name is invalid. Must begin with : test")
+   }
+}
+
+func (c *Client) Insert(database string, collection string, doc map[string]string) error {
+    VerifyITCollection(collection)
```

Comment:
Verification should be removed from Insert function and all rest functions

Reply:
Hello @GhMartingit , can you please elaborate the point. or if you have any approach in your mind, please share. Thanks